repositories/dbmodels/db_decision.go

```diff
@@ -23,6 +23,7 @@ type DbDecision struct {
 	CreatedAt            time.Time   `db:"created_at"`
 	Outcome              string      `db:"outcome"`
 	ScenarioId           string      `db:"scenario_id"`
+	ScenarioIterationId  pgtype.Text `db:"scenario_iteration_id"`
```

Comment:
IIUC, this should be `string` as soon as we have backfilled existing decision

Reply:
yes